pix-editor/app/services/storage.js

```diff
@@ -10,15 +10,18 @@ export default class StorageService extends Service {
 
   async uploadFile({ file, filename, date = Date, isAttachment = false }) {
     filename = filename || file.name;
-    const url = this.config.storagePost + date.now() + '/' + encodeURIComponent(filename);
     return this._callAPIWithRetry(async (token) => {
       const headers = {
         'X-Auth-Token': token,
         'Content-Type': file.type,
       };
+      let finalName = filename;
```

Comment:
Peut être même en profiter pour déplacer la ligne 12 qui est pas top car on réassigne un argument:
```suggestion
      let finalName = filename ||| file.name;
```